frontend/pages/read_upload.tsx

```diff
@@ -41,21 +53,31 @@ const InputPanel = (props: InterfacePanelProps) => {
   );
 };
 
-const PlayButton = () => (
-  <button className="flex justify-center items-center w-48 h-48 rounded-full bg-papyrus-200 hover:bg-papyrus-300 focus:outline-none">
-    <div className="ml-4 w-0 h-0 border-b-transparent border-l-brown-900 border-t-transparent border-r-transparent border-solid border-t-[3rem] border-r-0 border-b-[3rem] border-l-[6rem]"></div>
-  </button>
-);
-
 interface ResultsPanelProps {
   fileName: string;
   fileText: string;
+  uploadStatus: string;
 }
 const ResultsPanel = (props: ResultsPanelProps) => {
+  const PlayButton = () => (
+    <button
+      className="flex justify-center items-center w-48 h-48 rounded-full bg-papyrus-200 hover:bg-papyrus-300 focus:outline-none"
+      onClick={async () => {
+        if (props.uploadStatus != "Press play to listen.") {

```

Comment:
Don't use string comparisons because they are easy to cause errors with using typos (they are not best practice). Declare enums instead.